### PR TITLE
U1 login navegation

### PR DESCRIPTION
**Pull request by lauuta185**
### Summary
In this card, i add a wolox link page in the app footer and I made the login and register buttons lead to a blank activity.
### Card
[[U1] Maquetado: Login - Navegación](https://trello.com/c/liUXWtu9/25-u1-maquetado-login-navegaci%C3%B3n)

### CODE REVIEW

File: app/src/main/java/com/example/wnews/LoginActivity.kt [hunk 1]
```diff
@@ -94,6 +102,14 @@ class MainActivity : AppCompatActivity() {
         return sharedPreferences.getString(PASSWORD, "")
     }
 
+    fun openWebPage(url: String) {
+        val webpage: Uri = Uri.parse(url)
+        val intent = Intent(Intent.ACTION_VIEW, webpage)
+        if (intent.resolveActivity(packageManager) != null) {
+            startActivity(intent)
+        } else println("hola")
```
Comment (bdecorneliis) [inline app/src/main/java/com/example/wnews/LoginActivity.kt]: Bye 👀

File: app/src/main/java/com/example/wnews/LoginActivity.kt [hunk 2]
```diff
@@ -33,13 +35,20 @@ class MainActivity : AppCompatActivity() {
...
         setObserve()
 
+        binding.signupButton.setOnClickListener {
+            with(Intent(this, SignUpActivity::class.java)) {
+                startActivity(this)
+            }
+        }
+
+        binding.footer.setOnClickListener{
+            openWebPage("https://www.wolox.com.ar")
```
Comment (bdecorneliis) [inline app/src/main/java/com/example/wnews/LoginActivity.kt]: Move the string values into string.xml!

File: app/src/main/java/com/example/wnews/LoginActivity.kt [hunk 3]
```diff
@@ -103,3 +119,7 @@ class MainActivity : AppCompatActivity() {
 
 }
 
+
+
+
+
```
Comment (bdecorneliis) [inline app/src/main/java/com/example/wnews/LoginActivity.kt]: Let's make your code more visually "elegant". Delete these unused lines...

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".Home">
+
+    <!-- TODO: Update blank fragment layout -->
```
Comment (bdecorneliis) [inline app/src/main/res/layout/fragment_home.xml]: Delete this commened line

**Review by bdecorneliis — CHANGES_REQUESTED** [review]
Some comment. 

_A good developer also delivers visually beautiful code..._
_-Me_

File: app/src/main/java/com/example/wnews/LoginActivity.kt (hunk 1 above)
Comment (lauuta185) [reply app/src/main/java/com/example/wnews/LoginActivity.kt]: Hahaha it's always good to remember where you came from here

**Pull request merged**